source/HiringPanel.cpp

```diff
@@ -108,7 +109,9 @@ void HiringPanel::Draw()
 
 bool HiringPanel::KeyDown(SDL_Keycode key, Uint16 mod, const Command &command, bool isNewPress)
 {
-	if(!player.Flagship())
+	if(command.Has(Command::HELP))
+		DoHelp("hiring", true);
+	else if(!player.Flagship())
```

Comment:
If you change your help control to "H" then this causes the game to crash if you don't have a flagship, as the code below to hire on an "H" press still runs.